framework/prefixrouter/module.go

```diff
@@ -80,31 +80,16 @@ func (m *Module) serve(
 
 		areas, _ := root.GetFlatContexts()
 		for _, area := range areas {
-			b, _ := area.Configuration.Get("flamingo.router.path")
-			baseURL, _ := b.(string)
 
-			// allow to override the baseurl
-			if baseurlVal, ok := area.Configuration.Get("prefixrouter.baseurl"); ok {
-				baseURL = baseurlVal.(string)
-			}
-
-			if strings.HasPrefix(baseURL, "/") {
-				baseURL = "scheme://host" + baseURL
-			}
-
-			if baseURL == "" {
-				m.logger.WithField("category", "prefixrouter").Warn("No baseurl configured for config area %v", area.Name)
-				continue
-			}
-
-			m.logger.WithField("category", "prefixrouter").Info("Routing ", area.Name, " at ", baseURL)
 			area.Injector.Bind((*flamingo.Logger)(nil)).ToInstance(m.logger.WithField("area", area.Name))
 			areaRouter := area.Injector.GetInstance(web.Router{}).(*web.Router)
```

Comment:
We need to adjust this: as this is initialized after config.GetFlatContexts() we will fallthrough here, and child config areas will get their parents config. We need to fix that.